tools/annotation_tools/turk_with_s3/create_jobs.py

```diff
@@ -74,29 +83,58 @@ def create_turk_job(xml_file_path: str, tool_num: int, input_csv: str):
             curr_question = question.format(query_params)
 
             print(curr_question)
-
-            new_hit = mturk.create_hit(
-                Title="CraftAssist Instruction Annotations",
-                Description="Given a sentence, provide information about its intent and highlight key words",
-                Keywords="text, categorization, quick",
-                Reward="1.0",
-                MaxAssignments=1,
-                LifetimeInSeconds=600,
-                AssignmentDurationInSeconds=600,
-                AutoApprovalDelayInSeconds=14400,
-                Question=curr_question,
-            )
+            if use_sandbox:
+                new_hit = mturk.create_hit(
+                    Title="CraftAssist Instruction Annotations",
+                    Description="Given a sentence, provide information about its intent and highlight key words",
+                    Keywords="text, categorization, quick",
+                    Reward="0.3",
+                    MaxAssignments=1,
+                    LifetimeInSeconds=600,
+                    AssignmentDurationInSeconds=600,
+                    AutoApprovalDelayInSeconds=14400,
+                    Question=curr_question
+                )
+            else:
+                new_hit = mturk.create_hit(
+                    Title="CraftAssist Instruction Annotations",
+                    Description="Given a sentence, provide information about its intent and highlight key words",
+                    Keywords="text, categorization, quick",
+                    Reward="0.3",
```

Comment:
should we use a config file/command line arg for those job related params?

Reply:
You can certainly consider that for the future, though these configs are rarely changed (unlike dashboard in Turk where we were experimenting) so I would not consider that highest pri